Hector101/e-Document-Manager
# Chore/149784367/test routes

### What does this PR do?
Run the test for all the end points and models

#### Description of Task to be completed?
setup test environment and test both database model and controllers

#### How should this be manually tested?
using the terminal, run ""npm test" for the test to run, And check the test coverage on the terminal.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Testing routes and models

#### Screenshots (if appropriate)
<img width="722" alt="screen shot 2017-08-04 at 10 16 15 am" src="https://user-images.githubusercontent.com/10426118/28962587-0804c28a-78fe-11e7-99e9-e29584371a81.png">

#### Questions:
N/A
